feat: Add note about MinK system accounts (#6115)
Kong/docs.konghq.com: app/konnect/mesh-manager/service-mesh.md

1. Click **New Control Plane** and complete the fields as needed.
1. Select the control plane you just created, and then click **Zones** in the sidebar.
1. Click **Create Zone**, configure the fields as needed, and follow the steps in the wizard to connect your zone to {{site.konnect_short_name}}.
    {:.note}

>     > **Note:** Mesh Manager automatically creates a special [managed service account](/konnect/org-management/system-accounts/) that is only used in the zone creation process. This special, read-only managed service account cannot be edited or deleted manually. Instead, it is removed by {{site.konnect_short_name}} after the zone is created.

Reviewer: ```suggestion
    > **Note:** Mesh Manager automatically creates a [managed service account](/konnect/org-management/system-accounts/) that is only used in the zone creation process. This managed service account cannot be edited or deleted manually. Instead, its life-cycle is managed by {{site.konnect_short_name}} and it is deleted automatically when the zone is deleted.
```
